app/presenters/publishing_api/role_appointment_presenter.rb

```diff
@@ -41,5 +35,15 @@ def links
         ],
       }
     end
+
+  private
+
+    def title
+      "#{item.person.name} - #{item.role.name}"
+    end
+
+    def locale
+      I18n.locale.to_s
```

Comment:
Maybe it's clearer if this is always `en`? I don't think we'll ever publish non-English role appointments.

Reply:
I think roles and people can have translations, but not the appointment?  (There's no locale for role_appointment in the whitehall schema).  It won't hurt very much either way I think